googletest/src/gtest-port.cc

```diff
@@ -83,7 +83,7 @@
 #include "gtest/gtest-message.h"
 #include "gtest/internal/gtest-internal.h"
 #include "gtest/internal/gtest-string.h"
-#include "src/gtest-internal-inl.h"
+#include "gtest-internal-inl.h"
```

Comment:
How is this related to the bazel change? Please revert and just leave the bazel related changes.